salt/cloud/clouds/digital_ocean.py

```diff
@@ -720,20 +772,18 @@ def destroy(name, call=None):
     return node
 
 
-def create_dns_record(hostname, ip_address):
+def post_dns_record(dns_domain, name, record_type, record_data):
```

Comment:
That's all fine. Just figured I would point it out. As long as the correct deprecation warnings are in place, that is fine to change.